scripts/main.py

```diff
@@ -33,9 +33,10 @@ def check_openai_api_key():
 
 
 def attempt_to_fix_json_by_finding_outermost_brackets(json_string):
-    if cfg.speak_mode and cfg.debug_mode:
-      speak.say_text("I have received an invalid JSON response from the OpenAI API. Trying to fix it now.")
-    logger.typewriter_log("Attempting to fix JSON by finding outermost brackets\n")
+    if cfg.debug_mode:
+        if cfg.speak_mode:
+            speak.say_text("I have received an invalid JSON response from the OpenAI API. Trying to fix it now.")
```

Comment:
This is going in the wrong direction.
`if a and b:` is pythonic, nested isn't.

Reply:
More pythonic and following PEP 8 would look like this, making it easier to read and maintain.
```python
    if cfg.debug_mode:
        message = "I have received an invalid JSON response from the OpenAI API. Trying to fix it now."
        log_message = "Attempting to fix JSON by finding outermost brackets\n"

        if cfg.speak_mode:
            speak.say_text(message)
        
        logger.typewriter_log(log_message)
```